developer/src/kmcmplib/src/meson.build

```diff
@@ -24,12 +25,10 @@ endif
 name_suffix = []
 
 if cpp_compiler.get_id() == 'emscripten'
-  links += ['-lnodefs.js', '-sMODULARIZE', '-sEXPORT_ES6', '--whole-archive', '--bind', '-sEXPORTED_RUNTIME_METHODS=[\'UTF8ToString\']']
```

Comment:
I gather that `-lnodefs.js` is basically "**l**ink **node** '**fs**'"?  Hence the PR title and why it's removed, while the other options and parameters are maintained.

Reply:
[`nodefs`](https://emscripten.org/docs/api_reference/Filesystem-API.html#nodefs) is an emscripten library that wraps NodeJS's `fs` library, yes.